lispyville.el

```diff
@@ -665,6 +665,74 @@ ARG has the same effect."
       (lispy--normalize-1))
     (goto-char orig-pos)))
 
+(evil-define-operator lispyville-comment-or-uncomment (beg end type)
+  "Like `evilnc-comment-operator' but handles unmatched delimiters.
+May insert newlines where necessary in order to move delimiters outside of the
+commented regions and preserve the balanced structure. "
+  (interactive "<R>")
+  (when (string-match-p (format "[%s]+$" (regexp-quote comment-start))
+                        (buffer-substring beg end))
+    (setq end (point-at-eol)))
+  (let* ((end-marker (copy-marker (1+ end))))
+    (save-excursion
+      (dolist (r (lispy--find-safe-regions beg end))
+        (comment-or-uncomment-region
+         (car r)
+         ;; must include trailing whitespace to avoid error with undo-tree
+         (progn (goto-char (cdr r))
+                (skip-chars-forward "[:space:]") (point)))))
+    (delete-trailing-whitespace beg (marker-position end-marker)))
```

Comment:
`end-marker` should work fine by itself I believe. `marker-position` should not be necessary.